<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>medicinesearch</class>
 <widget class="QWidget" name="medicinesearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="editName">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>113</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>药品名称</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSearch">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>50</y>
     <width>106</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblImage">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>140</y>
     <width>301</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string>medicineImage</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>581</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
